packages/http-client/src/remote-admin-api.ts

```diff
@@ -86,9 +106,23 @@ export class RemoteAdminApi implements AdminApi {
         )}`,
       },
     })
-    return response.models.map((modelStreamIDString: string) => {
-      return StreamID.fromString(modelStreamIDString)
+    return response.models.map((id) => StreamID.fromString(id))
```

Comment:
if we rename `indices` to `indexedModels` (or similar) and deprecate `models` in the output, we should update this to get the streamids from the non-deprecated field to future-proof it.

Reply:
ping.  We should prefer reading from the `modelData` field in the response and only fall back to reading from `models` if `modelData` is missing.  That way in the future we can eventually remove `models` from the response entirely without breaking updated clients